build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -eo pipefail
+
+PACKAGE_NAME=$(awk -F' = ' '{gsub(/"/,"");if($1=="name")print $2}' pyproject.toml)
+VERSION=$(poetry version -s)
+
+ROOT_PATH="$PWD"
+ZIP_PATH="$ROOT_PATH/dist/$PACKAGE_NAME-$VERSION.zip"
+
```

Comment:
Can you just add the comment here for how to install the bundle plugin for poetry

Reply:
Added in [66aac0a](https://github.com/podaac/swodlr-raster-create/pull/1/commits/66aac0a4e746a1b1aeac78e6d9e3689d218ca361)